packages/@okta/vuepress-theme-prose/components/SidebarItem.vue

```diff
@@ -97,11 +99,26 @@ export default {
     this.setData();
   },
   watch: {
-    
     link() {
       this.setData();
     },
 
+    // Will triggers when StackSelector component will change it value.
+    "stackSelectorData.to"() {
+      // After StackSelector value has changed, route link will be modified.
+      // This condition will be true only for SidebarItems that contains links on pages with StackSelector.
+      if (this.link.frameworks && this.link.frameworks.length) {
```

Comment:
I noticed one small thing here -- we can directly use `Array.isArray` instead of this series of checks

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray